javagen/src/main/java/com/azure/autorest/model/clientmodel/EnumType.java

```diff
@@ -96,4 +103,77 @@ public final String validate(String expression) {
     public String toString() {
         return getName();
     }
+
+    public static class Builder {
```

Comment:
Converted to Builder.

Added `elementType`, default as ClassType.String. (not sure about naming, will `enumValueType` better?)